src/render-html/messageHeaderAsHtml.js

```diff
@@ -25,7 +25,7 @@ export default ({ item, subscriptions, auth, narrow, doNarrow }) => {
   data-narrow="${topicNarrowStr}"
   data-msg-id="${item.id}"
 >
-  ${item.subject}
+  ${item.match_subject || item.subject}
```

Comment:
@borisyankov thoughts on this. 
<img width="306" alt="screen shot 2018-02-15 at 8 11 27 pm" src="https://user-images.githubusercontent.com/18511177/36262476-8a0102c0-128d-11e8-9769-1cc354815e34.png">

improving search functionality. And rendering in webView had made lot easier to implement it :)